framework/src/main/java/org/checkerframework/framework/source/SourceChecker.java

```diff
@@ -2530,4 +2535,33 @@ private void printGitProperties() {
             System.out.println("IOException while reading git.properties: " + e.getMessage());
         }
     }
+
+    /**
+     * Extract the version of the Checker Framework
+     *
+     * @throws IOException if unable to read docs/developer/release/release.properties or
+     *     docs/examples/MavenExample/pom.xml file
+     * @throws NullPointerException if tag present in release.properties for Checker Framework
+     *     version is not found in docs/examples/MavenExample/pom.xml
+     * @return Checker Framework version {@link String}
+     */
+    private String getCheckerVersion() throws IOException {
+        String version = null;
+        String RLS_FILE = "/docs/developer/release/release.properties";
+        Properties releaseProperties = getProperties(getClass(), RLS_FILE);
+        String startTag = releaseProperties.getProperty("checkers.ver.0");
+        String endTag = releaseProperties.getProperty("checkers.ver.1");
+        String XML_FILE = "/docs/examples/MavenExample/pom.xml";
```

Comment:
Does this work?  It crashed for me when I tried to run it, because neither of the two files is not available in `checker.jar`.

Reply:
You could use `git.properties` instead.  See method `printGitProperties`.